README.md

```diff
@@ -467,14 +467,32 @@ With the use of the [Home Assistant RESTful][ha-rest] integration, one should
 be able to grab some interesting data from this add-on.
 
 Example:
-
+Set the port in the add-on settings; for example: 6789 and your IP of your Home Assistant instance; the hassio.local didn't work for me.
+Add a rest sensor:
 ```yaml
-sensor:
-  - platform: rest
-    resource: http://a0d7b954-wireguard
+platform: rest
+name: wireguard_sensor
+resource: http://192.168.3.72:6789/
```

Comment:
```suggestion
resource: http://a0d7b954-wireguard
```

Reply:
Is this http://a0d7b954-wireguard when you don't give a port to the status API?
How is a0d7b954 generated? 

If that works that's great

Reply:
Without specifying a port in the `http` protocol, automatically assumes port `80`, which is the port it is running on.

`a0d7b954` is a unique identifier for the add-on repository, in this case `a0d7b954` is the Community Add-ons stable repository. This is always the same for all users.

Reply:
@frenck Could you please explain what does the address itself account for? How can you reach the API on that address? The only way I can access the API is by giving it a port and then requesting on whatever the local address of my HA is running + the port I give to the Wireguard add-on.